internal/azure/access_whitelisting/access_whitelisting.feature

```diff
@@ -7,25 +7,41 @@ Feature: Object Storage Has Network Whitelisting Measures Enforced
 
   #Rule: CHC2-SVD030 - protect cloud service network access by limiting access from the appropriate source network only
 
-    @s-azaw-001
-    Scenario: Check Object Storage is Configured With Network Source Address Whitelisting
-      Given the CSP provides a whitelisting capability for Object Storage containers
-      And a specified azure resource group exists
-      When we examine the Object Storage container in environment variable "TARGET_STORAGE_CONTAINER"
-      Then whitelisting is configured with the given IP address range or an endpoint
+    Background:
+      Given an Azure subscription is available
+      And azure resource group specified in config exists
 
     @s-azaw-002
     Scenario Outline: Prevent Object Storage from Being Created Without Network Source Address Whitelisting
-      Given security controls that Prevent Object Storage from being created without network source address whitelisting are applied
-      And a specified azure resource group exists
-      When we provision an Object Storage container
-      And it is created with whitelisting entry "<Whitelist Entry>"
-      Then creation will "<Result>"
+      Then creation of a storage account with "<Whitelist Entry>" whitelisting entry "<Result>"
```

Comment:
Whitelist/Blacklist are banned terms in places like the Linux Kernel, we should probably follow their example.

Perhaps this alternative?
```
    Scenario Outline: Prevent Object Storage from Being Created Without Network Source Addresses Allowed
    Then creation of a storage account <Result> with allowed address <Entry>
```

Reply:
Great point. In addition to the requested change, I'm also renaming the probe and feature files.